js/apps/account-ui/src/routes.tsx

```diff
@@ -72,6 +70,8 @@ export const RootRoute: RouteObject = {
     ResourcesRoute,
     ContentRoute,
   ],
+  element: <Root />,
+  errorElement: <ErrorPage />,
```

Comment:
Just a small nit, but let's leave these properties on top where they used to be.